week1/3-website/index.html

```diff
@@ -16,7 +17,90 @@
+          </h2>
+          <p> <ol>
+             <li>Keep regular sleep hours
+                Going to bed when you feel tired and getting up at roughly the same time helps teach your body to sleep better. Try to avoid napping where possible.<br>
+              </li><li>Create a restful environment
+               Dark, quiet and cool environments generally make it easier to fall asleep and stay asleep. Watch our video for tips on how to sleep better.<br>
+              </li><li>Move more, sleep better
+               Being active can help you sleep better. These videos can get you going, but remember to avoid vigorous activity near bedtime if it affects your sleep.<br>
+              </li><li>Confront sleeplessness
+               If you are lying awake unable to sleep, do not force it. Get up and do something relaxing for a bit, and return to bed when you feel sleepier.<br>
+              </li><li>Write down your worries
+             If you often lie awake worrying about tomorrow, set aside time before bed to make a list for the next day. This can help put your mind at rest.<br>
+            </li><li>Put down the pick-me-ups
+             Caffeine and alcohol can stop you falling asleep and prevent deep sleep. Try to cut down on alcohol and avoid caffeine close to bedtime.<br>
+            </li> </ol>
+            </p>
+
+
+        </article>
+        
+         
+         <article id="calories" class="article">
+          <h2 >How Your Body Uses Calories While You Sleep</h2>
+        <p>
+           During the night, your body goes to work repairing any damage done on a cellular level during your waking hours. For instance, if you exercise during the day, your muscles will recover and repair themselves at night, which requires energy. Food digestion also uses energy, as your body breaks down your meal into usable fuel for the following day.
+          </p>
+         </article>
+        
+        
+         <article id="science"class="article" >
+          <h2 >What the Science Says</h2>
+        <p>
+           Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam at erat ornare,<br>
+            bibendum nulla sed, efficitur mi. Morbi lobortis mi ipsum, et fermentum neque varius vel.<br> Interdum et malesuada fames ac ante ipsum primis in faucibus. Sed purus nulla, dignissim id nulla et, imperdiet aliquet metus. Aliquam luctus eleifend tellus in suscipit. Sed non eleifend augue, ac hendrerit erat. Cras vel massa a nulla sollicitudin lacinia at id nisl. Nullam molestie tellus eget venenatis finibus. Nullam sit amet ullamcorper metus. Vivamus quis semper enim, ornare maximus sem.
+        </p>
+
+       </article>
+       <img id="header-img" src="man-taking-a-nap.jpg"        alt="man taking a nap" />
+
+       </section>
+       
+         
+        
+      </main>
+  </div>
+      <footer>
+            by:Abdennour Hachemi                                           Mai 2020
+      </footer>
   </body>
```

Comment:
Muddled tags. Always close tags in the order they're opened- in this case, close the div before closing the body.